manifests/oned/sunstone/config.pp

```diff
@@ -20,6 +20,7 @@
   $listen_ip          = $one::params::sunstone_listen_ip,
   $enable_support     = $one::params::enable_support,
   $enable_marketplace = $one::params::enable_marketplace,
+  $tmpdir             = $one::params::sunstone_tmpdir,
```

Comment:
Please add the parameter tmpdir also to manifests/params.pp and set it to OpenNebula default (/var/tmp).